test/workers/find_and_suspend_inactive_accounts_worker_test.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class FindAndSuspendInactiveAccountsWorkerTest < ActiveSupport::TestCase
+  setup do
+    @accounts = {to_suspend: [], not_to_suspend: []}
+
+    # Provider with no traffic since 1 year ago and was created before that. To be suspended
```

Comment:
Instead of establishing `1.year.ago` or so manually, I'd like to change it to use the constants with these values from the Account states 😄